docs/api/v1.19.md

```diff
@@ -16,9 +16,7 @@ redirect_from:
      will be rejected.
 -->
 
-# Docker Engine API v1.19
-
-## 1. Brief introduction
+# 1. Brief introduction
```

Comment:
Hah, okay. I assumed it was a bug because just the first heading was h2, everything else was h1. I have fixed this the non-lazy way now and updated all the headings to be the correct level. :)